core/src/main/java/org/commonjava/maven/ext/core/groovy/ASTTransformer.java

```diff
@@ -194,18 +206,11 @@ private void changeBaseScriptType(final AnnotatedNode parent, final ClassNode cN
 
         List<AnnotationNode> annotations = parent.getAnnotations( DEPRECATED_COMMAND_TYPE );
         if (cNode.getAnnotations( DEPRECATED_COMMAND_TYPE ).isEmpty()) { // #388 prevent "Duplicate annotation for class" AnnotationFormatError
-
-            // TODO : handle scenario where the annotation is not fully qualified which comes through as a variable
```

Comment:
Anything added for these?

Reply:
No, this code came from the original classes, so its just cleaning it up